File: platform/sdk/python/examples/graph/graph-012_spark_roundtrip_2k_3k.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from xuanji_sdk_graph import GraphClient

client = GraphClient({"endpoint": "graph.local"})
result = client.sparkRoundtrip2k3k()
if not result["ok"] or result["consistency"] != "verified":
    sys.exit(1)
print("XJ-OK: graph-012_spark_roundtrip_2k_3k")
sys.exit(0)

File: platform/sdk/nodejs/examples/graph/graph-012_spark_roundtrip_2k_3k.js
const { GraphClient } = require("../../mox-sdk-graph");
const client = new GraphClient({ endpoint: "graph.local" });
const result = client.sparkRoundtrip2k3k();
if (!result.ok || result.consistency !== "verified") process.exit(1);
console.log("XJ-OK: graph-012_spark_roundtrip_2k_3k");
process.exit(0);
